ts/src/bitrue.ts

```diff
@@ -863,53 +1264,89 @@ export default class bitrue extends Exchange {
          * @method
          * @name bitrue#fetchTicker
          * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+         * @see https://github.com/Bitrue-exchange/Spot-official-api-docs#24hr-ticker-price-change-statistics
+         * @see https://www.bitrue.com/api-docs#ticker
+         * @see https://www.bitrue.com/api_docs_includes_file/delivery.html#ticker
          * @param {string} symbol unified symbol of the market to fetch the ticker for
          * @param {object} [params] extra parameters specific to the bitrue api endpoint
          * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const uppercaseBaseId = this.safeStringUpper (market, 'baseId');
-        const uppercaseQuoteId = this.safeStringUpper (market, 'quoteId');
-        const request = {
-            'currency': uppercaseQuoteId,
-            'command': 'returnTicker',
-        };
-        const response = await this.klinePublicGetPublicCurrencyJson (this.extend (request, params));
+        let type = undefined;
+        [ type, params ] = this.handleMarketTypeAndParams ('fetchTicker', market, params);
+        let subType = undefined;
+        [ subType, params ] = this.handleSubTypeAndParams ('fetchTicker', market, params);
```

Comment:
@sc0Vu When the symbol is always known and the exchange does not have conflicting ids (which does not seem to be the case) we don't need to use HandleMarketType/SubType, you can do
`market['type']` directly
![image](https://github.com/ccxt/ccxt/assets/43336371/b5c0aed4-887e-42e8-a168-7286b4dbc155)